src/languages/en.js

```diff
@@ -552,12 +559,28 @@ export default {
         error: {
             firstNameLength: 'First name shouldn\'t be longer than 50 characters',
             lastNameLength: 'Last name shouldn\'t be longer than 50 characters',
-            characterLimit: ({limit}) => `Exceeds the max length of ${limit} characters`,
             hasInvalidCharacter: ({invalidCharacter}) => `Please remove the ${invalidCharacter} from the name field.`,
             comma: 'comma',
             semicolon: 'semicolon',
         },
     },
+    privatePersonalDetails: {
+        personalDetails: 'Personal details',
+        privateDataMessage: 'These details are used for travel and payments. They are never shown on your public profile.',
+        legalName: 'Legal name',
+        legalFirstName: 'Legal first name',
+        legalLastName: 'Legal last name',
+        dateOfBirth: 'Date of birth',
+        homeAddress: 'Home address',
+        error: {
+            hasInvalidCharacter: ({characterName}) => `Please remove the ${characterName} from the field above.`,
+            legalFirstNameEmpty: 'Please enter a valid legal first name',
+            legalLastNameEmpty: 'Please enter a valid legal last name',
+            dateInvalid: 'Please enter a valid date.',
+            dateTooRecent: ({numYears}) => `Date needs to be less than ${numYears} years ago.`,
+            dateTooOld: ({numYears}) => `Date cannot be more than ${numYears} years ago.`,
```

Comment:
@shawnborton sorry for tagging you all over recently, would you mind looking at these error messages to see if they make sense for our upcoming Legal name page & Date of birth page?

Maybe I can get rid of the trailing `.` since that's inconsistent here, otherwise are they decent?